README.md

```diff
@@ -240,81 +249,60 @@ By using the `--live` flag you can stay focused on API design (OpenAPI or AsyncA
+### The bump overlay command
 
-The Overlay specification of OpenAPI makes it possible to modify the content of an API definition file by adding a layer on top of it. That layer helps add, remove, or change some or all of the content of the original definition.
+The [Overlay specification](https://github.com/OAI/Overlay-Specification) from the OpenAPI Initiative makes it possible to modify the content of an API definition by adding a layer on top of it. That layer helps adding, removing or changing some or all of the content of the original definition. 
 
-Technically, the `bump overlay` command will output a modified version of the `[DEFINITION_FILE]` (an OpenAPI or AsyncAPI document) by applying the operations described in the `[OVERLAY_FILE]` Overlay file to the original API document.
-
-To redirect the output of the command to a new file you can run the following:
+The `bump overlay` command takes an original API document, applies the changes in the overlay document, and outputs a modified version. No changes are made directly to the original document.
 
 ```shell
-bump overlay api-document.yaml overlay-file.yaml > api-overlayed-document.yaml
+bump overlay openapi.yaml overlay.yaml
 ```
 
-_**Note:** you can also apply the overlay during the [`bump deploy` command]((#bump-deploy-file)) with the new `--overlay` flag:_
+To redirect the output of the command to a new file you can run:
 
 ```shell
-bump deploy api-document.yaml --doc my-doc --token my-token --overlay overlay-file.yaml
+bump overlay openapi.yaml overlay.yaml > modified-openapi.yaml
 ```
 
-## Development
-
-Make sure to have Node.js (At least v20) installed on your machine.
-
-- Install node dependencies with
-
-  ```sh-session
-  npm install
-  ```
+Currently overlays only support OpenAPI, not AsyncAPI.
 
-- Compile the Typescript code
+You can also apply the overlay using the [`deploy` command](#bump-deploy-file) with the `--overlay` flag:
 
-  ```sh-session
-  npm run build
-  npm run clean # to remove build artifacts
-  ```
-
-- Format the codebase to comply with the linter rules
-
-  ```sh-session
-  npm run fmt
-  ```
+```shell
+bump deploy openapi.yaml --doc my-doc --token my-token --overlay overlay.yaml
+```
 
-- Run the test suites
+IF there are multiple overlays which need to be applied, the --overlay can be passed multiple times.
 
-  ```sh-session
-  npm run test
-  npm run test-coverage # Run tests with coverage
-  ```
+```shell
+bump deploy openapi.yaml \
+  --doc my-doc \
+  --token my-token \
+  --overlay overlay1.yaml \
+  --overlay overlay2.yaml
+```
 
-## Compatible specification types
+## Compatible API description formats
 
-We currently support [OpenAPI](https://github.com/OAI/OpenAPI-Specification) from 2.0 (called Swagger) to 3.1 and [AsyncAPI 2.x](https://www.asyncapi.com/docs/reference/specification/latest) specification file types. Both YAML and JSON file formats are accepted file inputs to the CLI.
```

Comment:
I think keeping Swagger is good for SEO, would you mind keeping it?

Reply:
The usual rule of thumb is to have it once at the top and then don't mention it again afterwards. We do have it once at the top and so I've just removed the second mention. Can pop it back if you prefer.